src/modules/checkIsValidUserInput.js

```diff
@@ -9,7 +9,12 @@
  * @return {boolean} - True if the user input is valid, false otherwise
  */
 function checkIsValidUserInput(userInput) {
-  /* Write your code here */
+  const isFourDigits = userInput.length === 4;
+  const hasUniqueDigits = new Set(userInput.split('')).size === 4;
+  const isNumber = !isNaN(+userInput);
```

Comment:
The check `isNumber` using `!isNaN(+userInput)` might not catch all invalid cases, such as strings with non-digit characters. Consider using a regular expression to ensure the input consists only of digits.